verilog/CST/verilog_matchers_test.cc

```diff
@@ -56,6 +56,24 @@ TEST(VerilogMatchers, MacroCallIdLeafTests) {
     verible::matcher::RunRawMatcherTestCase<VerilogAnalyzer>(test);
 }
 
+// Tests for SymbolIdentifierLeaf matching
+TEST(VerilogMatchers, SymbolIdentifierLeaf) {
+  const RawMatcherTestCase tests[] = {
+    // TODO cases
+      {SymbolIdentifierLeaf(), "", 0},
+      {SymbolIdentifierLeaf(), EmbedInClass(""), 1},
+      {SymbolIdentifierLeaf(), EmbedInClassMethod("`uvm_foo"), 3},
+      {SymbolIdentifierLeaf(), "`uvm_foo(\"foo\");", 2},
+      {SymbolIdentifierLeaf(), "`uvm_foo(\"foo\")\n", 2},
+      {SymbolIdentifierLeaf(), EmbedInClassMethod("`uvm_foo(\"foo\");"), 4},
+      {SymbolIdentifierLeaf(), EmbedInClassMethod("`uvm_foo(\"foo\")\n"), 4},
+      {SymbolIdentifierLeaf(), EmbedInClassMethod("uvm_foo(\"foo\");"), 4},
+      {SymbolIdentifierLeaf(), EmbedInClassMethod("$uvm_foo(\"foo\");"), 4},
+  };
+  for (const auto& test : tests)
+    verible::matcher::RunRawMatcherTestCase<VerilogAnalyzer>(test);
```

Comment:
*sigh* I never got around to cleaning up this entire test file to use a substring-tagged annotation like `SyntaxTreeSearchTestCase`.  Huge TODO for us.

Reply:
There’s a lot of work to do 😂. For now I’ll leave this as it is to make the file uniform.